image_width: 1280
image_height: 720
camera_name: imx477__base_soc_i2c0mux_i2c_0_imx477_1a_1280x720
camera_matrix:
  rows: 3
  cols: 3
  data: [330.14795521, 0.0, 653.24626757, 0.0, 330.18781026, 364.70683488, 0.0, 0.0, 1.0]
distortion_model: equidistant
distortion_coefficients:
  rows: 1
  cols: 4
  data: [-0.01084982, -0.00627448, 0.00488222, -0.00182596]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
projection_matrix:
  rows: 3
  cols: 4
  data: [330.14795521, 0.0, 653.24626757, 0.0, 0.0, 330.18781026, 364.70683488, 0.0, 0.0, 0.0, 1.0, 0.0]
